plugins/by-name/molten/default.nix

```diff
@@ -80,9 +80,11 @@ mkVimPlugin {
         [
           "none"
           "image.nvim"
+          "wezterm"
         ]
         ''
           How images are displayed.
+          wezterm require plugins.wezterm.enable = true
```

Comment:
This feels a little clunky. I think the indended wording is something like:

> ``If set to `"wezterm"`, `plugins.wezterm.enable` must be true.``

I'd also have a blankline (to trigger a paragraph break)

---

However it feels strange to have this notice for wezterm but not image.nvim? Maybe the original description was fine as it was?

Reply:
idk because image.nvim directly refer to a plugin but not wezterm. But it's as you want, I don't really care